tests/data/mongod.conf

```diff
@@ -0,0 +1,15 @@
+net:
+  bindIp: localhost,10.126.63.237
```

Comment:
I can't see this IP defined anywhere else - is this related to your local config?

Reply:
Great question, this IP actually gets overwritten in [line 87](https://github.com/MiaAltieri/mongodb-operator/blob/71559c44d1a0611f7d33f9ed080365ee2c64e3f2/tests/integration/test_charm.py#L87) this is because this IP is meant to be the unit's IP address and we wouldn't know it ahead of time. I see how this is confusing. Maybe I could change it to:
```
net:
  bindIp: localhost,<placeholder_unit_ip>
```
Would that make things more clear?

Reply:
Ah, I understand now. Setting it to explicitly be a placeholder would make this clearer, thanks!